Source/Scene/ShadowMap.js

```diff
@@ -385,6 +379,40 @@ define([
         },
 
         /**
+         * Determines if a normal bias will be applied to shadows.
+         *
+         * @memberof ShadowMap.prototype
+         * @type {Boolean}
+         * @default true
+         */
+        normalOffset : {
+            get : function() {
+                return this._normalOffset;
+            },
+            set : function(value) {
+                this.dirty = this._normalOffset !== value;
+                this._normalOffset = value;
```

Comment:
This setter should also set `this._terrainBias.normalOffset`, `this._primitiveBias.normalOffset`, and `this._pointBias.normalOffset`,